fe/src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from './Login.module.scss';
+
+function Login(): JSX.Element {
+    return (
+        <div className={styles.login}>
+            <div className={styles.loginItems}>
+                <h1 className={styles.h1}>Login</h1>
+                <input className={styles.input} type="text" placeholder="example@tptlive.ee" id="username"/>
+                <input className={styles.input} type="text" placeholder="password" id="password"/>
+                <button className={styles.button}>Login</button>
+            </div>
```

Comment:
Also here. We should use Chakra UI elements, you can google for some templates to base things upon.

Also there is no form validation, maybe this could work - https://chakra-ui.com/docs/form/form-control